src/store/create-conversation/saga.ts

```diff
@@ -56,6 +56,7 @@ export function* createConversation(action) {
 
 export function* saga() {
   yield fork(authWatcher);
+  yield fork(handleCreateConversation);
```

Comment:
I'm guessing a rebase or merge from main will get rid of this but just wanted to point it out to double check.

Reply:
Yeah there were a few conflicts to fix up. All sorted :) Shall merge this tomorrow.